dicoogle/src/main/java/pt/ua/dicoogle/server/users/UsersStruct.java

```diff
@@ -1,40 +1,39 @@
 /**
  * Copyright (C) 2014  Universidade de Aveiro, DETI/IEETA, Bioinformatics Group - http://bioinformatics.ua.pt/
- *
+ * <p>
```

Comment:
Please try to remove these extraneous `<p>` tags. They do not conform with our current doc formatting conventions.